db/models/article.model.js

```diff
@@ -18,7 +17,61 @@ exports.selectArticle = (article_id) => {
   });
 };
 
-exports.selectAllArticles = async () => {
+exports.selectAllArticles = async (query) => {
+  const topicSafelist = [
```

Comment:
The topics safe list does work here, It might be a nice refactor if you get time later on to make a request to the DB to get this list incase any new topics were added.